src/hooks/useGlobalObjectEventListener.ts

```diff
@@ -11,7 +11,7 @@ import { warning } from "./warning";
  * @param {Window|Document} globalObject The global object to add event onto
  * @param {string} eventName The event to track
  * @param {Function} callback The callback to be called on event
- * @param {object} conditions The options to be passed to the event listener
+ * @param {boolean | {once?: boolean; passive?: boolean; signal?: AbortSignal; capture?: boolean;}} listenerOptions The options to be passed to the event listener
```

Comment:
@qqpann Is there a way to make this also import-able from src/types/utils? 🤔 I am not sure and wondering if you have an idea.

Reply:
Hmm, to my knowledge, we can write like this:

```ts
/**
  * @param {ListenerOptions} listenerOptions The options to be passed to the event listener
  */
```

but the actual type inside will be hidden by this name alias.

I think this is a matter of preference.

Reply:
But if we create an alias type already, it's cleaner to use it here as well IMO.

Reply:
Yeah this is fine. I want the type to be aliased, so if JSDOC type is hidden, that is fine.